src/go/rpk/pkg/cli/cmd/plugin/plugin.go

```diff
@@ -166,67 +177,80 @@ be overridden by specifying the --bin-dir flag.
 		Args: cobra.ExactArgs(1),
 		Run: func(_ *cobra.Command, args []string) {
 			name := args[0]
+			var (
+				autoComplete bool
+				body         []byte
+				err          error
+			)
+			if len(version) > 0 {
```

Comment:
I think this is fine for this PR, but will need some follow up if we are able to remove the version requirement. (followup issue / PR)